geps/gep-1619.md

```diff
@@ -484,22 +511,46 @@ standards due to wide acceptance are:
-name. However, certain cookie attributes require special consideration, which we will discuss in detail.
+Unless a `SessionPersistencePolicy` API field can be satisfied through a manipulating a cookie attribute, the attributes
+of the cookies are considered as opaque values in this spec and are to be determined by the individual implementations.
+Let's discuss some of these cookie attributes in more detail.
+
+#### Name
+
+The `Name` cookie attribute can be configured via the `SessionName` field on `SessionPersistencePolicy`. However, this
+field is considered extended support level. This is because some implementations, such as ones supporting global load
+balancers, don't have the capability to configure the cookie name.
+
+#### TTL
+
+The `TTL` cookie attribute may be influenced by the `AbsoluteTimeoutSeconds` field on `SessionPersistencePolicy`.
+However, it's important to understand that `AbsoluteTimeoutSeconds` represents the duration of the entire session, not
+just the cookie duration. Conversely, the cookie's `TTL` attribute does not have to be configured in order to implement
+`AbsoluteTimeoutSeconds`.
 
 #### Path
 
-The `Path` attribute should be automatically determined by implementors. When the `SessionPersistencePolicy` policy is
-attached to an xRoute, the implementor should interpret the path as configured on the xRoute. However, if the
-`SessionPersistencePolicy` policy is attached to a service, the path should be left unset.
+The cookie's `Path` attribute defines the URL path that must exist in order for the client to send the `cookie` header.
+Whether attaching session persistence to an xRoute or a service, it's important to consider the relationship the cookie
+`Path` attribute has with the route path.
+
+When the `SessionPersistencePolicy` policy is attached to an xRoute, the implementor should interpret the path as
+configured on the xRoute. To interpret the `Path` attribute from an xRoute, implementors should take note of the
+following:
 
-To interpret the `Path` attribute from an xRoute, implementors should take note of the following:
 1. For an xRoute that matches all paths, the `Path` should be set to `/`.
 2. For an xRoute that has multiple paths, the `Path` should be interpreted based on the route path that was matched.
 3. For an xRoute using a path that is a regex, the `Path` should be set to the longest non-regex prefix (.e.g. if the
    path is /p1/p2/*/p3 and the request path was /p1/p2/foo/p3, then the cookie path would be /p1/p2).
 
+It is also important to note that this design makes persistent session unique per route path. For instance, if two
+distinct routes, one with path prefix `/foo` and the other with `/bar`, both target the same service, the persistent
+session won't be shared between these two paths.
+
+Conversely, if the `SessionPersistencePolicy` policy is attached to a service, the `Path` attribute should be left
```

Comment:
I would add "It is recommended that each service uses a different cookie name, for cases where the client is a browser making use of multiple services in the same page"